src/integration-tests/test_admin_client.py

```diff
@@ -70,6 +71,53 @@ def post_n_msgs(
             posted[task.uri] = task
 
 
+def extract_stats(admin_response: str) -> dict:
+    """
+    Extracts the dictionary containing stats from the specified 'admin_response'.
+    Note that due to xsd schema limitations it's not possible to make a schema for a json
+    containing random keys. Due to this, the stats encoding is not ideal:
+    - The outer layer is a 'str' response from the admin session
+    - Next layer is a 'dict' containing "stats" field with 'str' text
+    - The last layer is another 'dict' containing the stats itself
+    """
+    d1 = json.loads(admin_response)
+    d2 = json.loads(d1["stats"])
+    return d2
+
+
+class GreaterThan:
```

Comment:
Add a small comment explaining the purpose of the `GreaterThan` class.

The class itself is not self-explanatory.